.github/CONTRIBUTING.md

```diff
@@ -108,6 +108,44 @@ With the server running as above (with `yarn start`), run this to open
 
     yarn run cy
 
+### Run AVA unit tests
+You can also run unit tests under `/test`:
+
+    yarn test
+
+#### Test Google Sheets API integration
+In order to test Google Sheets API integration workflow, you will need `test/fixtures/credentials.json` and `test/fixtures/token.json`.
+
+`credentials.json`
+
+```json
+{
+    "client_secret": YOUR_CLIENT_SECRET,
+    "client_id": YOUR_CLIENT_ID,
+    "redirect_uri": "http://localhost:5000"
+}
+```
+
+`token.json`: run `yarn test` to get the authentication URL e.g. ``. You need to authenticate yourself to get the code. Once you have the code,
```

Comment:
@yehee Maybe I'm missing something, but where is `credentials.json` actually _used_?